src/Plugin/authentication/OAuth2ServerAuthentication.php

```diff
@@ -84,7 +98,14 @@ protected function getOAuth2Info(RequestInterface $request) {
 
     $server = $plugin_definition['oauth2Server'];
     $scope = !empty($plugin_definition['oauth2Scope']) ? $plugin_definition['oauth2Scope'] : '';
-    return ['server' => $server, 'scope' => $scope];
+    $cc_user = !empty($plugin_definition['oauth2ClientCredentialsUser']) ? $plugin_definition['oauth2ClientCredentialsUser'] : '';
+    $cc_uid = !empty($plugin_definition['oauth2ClientCredentialsUid']) ? $plugin_definition['oauth2ClientCredentialsUid'] : '';
+    return [
+      'server' => $server,
+      'scope' => $scope,
+      'client_credentials_user' => $cc_user,
```

Comment:
Is there any impediment to make it camel case? If not, I'd rather camel case since it's consistent with the rest of the annotation keys.